force-app/main/default/classes/ProgramEngagementService_TEST.cls

```diff
@@ -52,6 +52,42 @@ public with sharing class ProgramEngagementService_TEST {
         bucketSelectorStub.assertCalledAsExpected();
     }
 
+    @IsTest
+    private static void shouldGetActiveStagesWithLabels() {
+        List<String> bucketNames = new List<String>{ 'Active' };
+
+        Schema.SObjectType programEngagementSObjType = ProgramEngagement__c.SObjectType;
+        Schema.SObjectField stageField = ProgramEngagement__c.Stage__c;
+
+        List<Bucket__mdt> stageBuckets = createBuckets();
```

Comment:
Is this remnants from another test, do we need buckets or stubbing buckets for this one?